packages/react-components/react-datepicker-compat/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@fluentui/react-datepicker",
+  "name": "@fluentui/react-datepicker-compat",
   "version": "9.0.0-alpha.0",
```

Comment:
**question:**  are we gonna go with 9 ? AFAIR the decision was to go with `0.x.x` ? to communicate "unstable" without need of using prerelease tags.